webrtc-stats.html

```diff
@@ -2646,6 +2649,43 @@ <h2>
                   Represents the total number of <a>RTCP SR</a> blocks sent for this <a>SSRC</a>.
                 </p>
               </dd>
+              <dt>
+                <dfn>roundTripTime</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Estimated round trip time for this <a>SSRC</a> based on the latest <a>RTCP Sender Report</a> (SR)
+                  that contains a <a>DLRR</a> report block as defined in [[!RFC3611]]. The Calculation of
+                  the round trip time is defined in section 4.5. of [[!RFC3611]]. If no SR contains the DLRR
+                  report block, or if the <a>delay since last RR</a> value in DLRR report blocks are all 0s,
+                  the round trip time is left undefined.
```

Comment:
```suggestion
                  the round trip time is defined in section 4.5. of [[!RFC3611]]. If the latest SR does not contain the DLRR
                  report block, or if the <a>delay since last RR</a> value in DLRR report blocks are all 0s,
                  the round trip time is left undefined.
```